camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/DslAndRebindYamlTest.java

```diff
@@ -61,58 +56,41 @@
 
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Lists;
-import com.google.common.io.Files;
 
 @Test
-public class DslAndRebindYamlTest extends AbstractYamlTest {
+public class DslAndRebindYamlTest extends AbstractYamlRebindTest {
```

Comment:
The `AbstractYamlRebindTest` does the rebind correctly - it makes sure the `mgmt` field is updated on rebind, so subsequent provisioning uses the new application context. Before this, `testDslSensorFromClass` was not doing what you'd expect - the second call returned the app from the first call to `doTestOnEntityWithSensor`, so the subsequent assertions weren't useful.